CHANGELOG.md

```diff
@@ -7,228 +7,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Allows users to set a max priority fee below suggested, just showing a warning in that case ([#11750](https://github.com/MetaMask/metamask-extension/pull/11750))
-- Rename effective gas price field to total gas fee ([#11754](https://github.com/MetaMask/metamask-extension/pull/11754))
-- Fix ipfs dependency vulernability ([#11745](https://github.com/MetaMask/metamask-extension/pull/11745))
-- Remove estimate detail above advanced gas controls in non-1559 network ([#11744](https://github.com/MetaMask/metamask-extension/pull/11744))
-- Fall back to aggregator name when icon is unavailable (Swaps Load View) ([#11718](https://github.com/MetaMask/metamask-extension/pull/11718))
-- EIP-1559 - Show minium native currency in banner when on testnets ([#11743](https://github.com/MetaMask/metamask-extension/pull/11743))
-- Fix a position of a tooltip icon ([#11739](https://github.com/MetaMask/metamask-extension/pull/11739))
-- Add gasPrice to be used on non-1559 networks for transaction details on confirm screen ([#11741](https://github.com/MetaMask/metamask-extension/pull/11741))
-- EIP-1559 - Provide better validation for gas price and gas limit ([#11736](https://github.com/MetaMask/metamask-extension/pull/11736))
-- EIP-1559 - Fix education modal typo ([#11740](https://github.com/MetaMask/metamask-extension/pull/11740))
-- Ensure that gas price in popover updates when api provided estimate updates ([#11727](https://github.com/MetaMask/metamask-extension/pull/11727))
-- EIP-1559 - Ensure transaction detail font-size and icon colors are consistent with Figma design ([#11735](https://github.com/MetaMask/metamask-extension/pull/11735))
-- EIP-1559 - Return null from GasTiming if on non-1559 network ([#11733](https://github.com/MetaMask/metamask-extension/pull/11733))
+### Added
+- [#12066](https://github.com/MetaMask/metamask-extension/pull/12066): Integrating new TokenRatesController to expand fiat conversion rates for some non-Mainnet networks
+- [#12110](https://github.com/MetaMask/metamask-extension/pull/12110): Add search tab in import tokens for custom networks
+- [#11798](https://github.com/MetaMask/metamask-extension/pull/11798): Add fake token warning and replacing `Add Token` labels with `Import Tokens`
+- [#11755](https://github.com/MetaMask/metamask-extension/pull/11755): Add `Refresh list` and `import tokens` link to the main page
+- [#11851](https://github.com/MetaMask/metamask-extension/pull/11851): Add Token Detection Toggle to Security & Privacy (Settings)
+- [#11845](https://github.com/MetaMask/metamask-extension/pull/11845): Add support for Swaps via a direct contract for wrapping and unwrapping
+- [#12042](https://github.com/MetaMask/metamask-extension/pull/12042): Add Max Fee Per Gas to transaction breakdown
+- [#11999](https://github.com/MetaMask/metamask-extension/pull/11999): Add eth_feeHistory to safe method list
```

Comment:
Maybe "Add eth_feeHistory to safe method list" should be "Add eth_feeHistory to API callable by dapps"

Reply:
updated